mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -60,7 +74,8 @@ def binary_cross_entropy(pred,
                          weight=None,
                          reduction='mean',
                          avg_factor=None,
-                         class_weight=None):
+                         class_weight=None,
+                         ignore_index=255):
```

Comment:
255 is so confusing

Reply:
In this function, we do not use ignore_index in the F.cross_entropy_loss.
So I refer to the setting of mmsegmentation, setting it to 255 by default.